baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -17,7 +17,7 @@
 // ** All changes to this file may be overwritten. **
 
 import * as gax from 'google-gax';
-import {APICallback, Callback, CallOptions, Descriptors, ClientOptions, LROperation, PaginationCallback, PaginationResponse} from 'google-gax';
```

Comment:
Hmmmm. Maybe it's better to start actually using these types if possible? :)  (in the client template, line 551 and nearby)